src/state/apis/nft/types.ts

```diff
@@ -32,7 +32,11 @@ export type NftItem = {
   description: string
   // TODO(gomes): store the collection data into nft[collectionAssetId] and make this `collectionId`
   // we can then join the item and collection, once the collection is in the store as well
-  collection: NftCollectionType
+  collectionId: AssetId
   medias: MediaUrl[]
   rarityRank: number | null
 }
+
+export type NftItemWithCollection = Omit<NftItem, 'collectionId'> & {
```

Comment:
Temporary type, revert before opening me

Reply:
Or perhaps not, so far it turns out to be useful as an intermediary value

Reply:
Keeping this as an intermediary type for parsers, but also as a return type for `selectNftItemsWithCollection`, to be used as a join selector for components to consume, so this can play nicely with `matchSorter`